tests/wipac_dev_tools_test.py

```diff
@@ -1,31 +0,0 @@
-"""Test the basic stuff about the package."""
```

Comment:
Thank you for deleting this, I'm not sure how this test made it this far :smile: